lib/msal-electron-proof-of-concept/src/appConfig/applicationConfiguration.ts

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+/**
+ * @type AuthOptions: Used to configure the auth options in the ApplicationConfiguration object.
+ * - clientId       - Client ID of your app registered with our Application registration portal : https://portal.azure.com/#blade/Microsoft_AAD_IAM/ActiveDirectoryMenuBlade/RegisteredAppsPreview in Microsoft Identity Platform
+ * - authority      - You can configure a specific authority, defaults to " " or "https://login.microsoftonline.com/common"
```

Comment:
>  [](start = 75, length = 1)

are you missing something here?

Reply:
Do you mean something like more config options? This is just the setup with a minimal amount of config options to test the "plugability" of the library into an Electron app. I will add the remaining auth options as their use cases come up during development. Or did you mean something else might be missing?

Reply:
The comment says "defaults to " " or …" is there something that's supposed to be in the blank quotes there? Or they don't have to specify an authority? I guess I don't see how the default can be " " or "https://login.microsoftonline.com/common", but I'm probably reading it wrong